<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基于NB-IoT的智能井盖检测系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>连接配置</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_state">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>501</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_state">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>服务器通讯状态 -&gt; 建立连接失败！！！</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_p1state">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>状态：</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_data">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>60</y>
        <width>501</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_p1data">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>数据流：</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_SubAll">
         <property name="text">
          <string>自动连接所有节点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_disconnect">
         <property name="text">
          <string>断开连接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Reconnect">
         <property name="text">
          <string>重新连接</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>组态</string>
     </attribute>
     <widget class="QLabel" name="label_p2n1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>节点1：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_p2n2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>节点2：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_p2n3">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>节点3：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
